build.gradle

```diff
@@ -182,11 +172,14 @@ configurations.all {
         // force testng dependency so we don't pick up a different version via GenomicsDB
         force 'org.testng:testng:' + testNGVersion
         force 'org.broadinstitute:barclay:' + barclayVersion
-        force 'com.twitter:chill_2.11:0.8.1'
+        force 'com.twitter:chill_2.12:0.10.0'
+        force 'org.apache.commons:commons-math3:3.5'
 
         // make sure we don't pick up an incorrect version of the GATK variant of the google-nio library
         // via Picard, etc.
         force googleCloudNioDependency
+
+        force 'com.esotericsoftware:kryo:4.0.0'
```

Comment:
Hm. I removed the reference to 5.0.3, since we've been testing all along with 4.0.0 due to this statement. The net effect is to move us from 3.0.3 to 4.0.0. I removed a couple of problematic "excludes" of "reflectasm'. They're pretty old (Adam is git-blamed) and we don't seem to need them (by seem, I mean tests pass). We might want to try at some point to move to 5.x of kryo, but I think that caused problems when I previously tried it, so I'll save that for another day.